frontend/src/component/playground/Playground/PlaygroundResultsTable/PlaygroundResultChip/PlaygroundResultChip.tsx

```diff
@@ -11,12 +11,14 @@ interface IResultChipProps {
     label: string;
     // Result icon - defaults to true
     showIcon?: boolean;
+    tabindex?: number;
```

Comment:
Allows you to make these non-tabbable. I don't think there is a reason they should be tabbable at all, so I might default it to -1 later.